README.md

```diff
@@ -30,7 +30,13 @@ Generally speaking, you will use this in combination with the [laminas/laminas-c
 
 ## Usage
 
+Create `.github/workflows/continuous-integration.yml` file with following content:
+
 ```yaml
+name: "Continuous Integration"
+
+on: [push, pull_request]
```

Comment:
I should probably explain why I used the following:

```yaml
on:
  pull_request:
  push:
    branches:
      - '[0-9]+.[0-9]+.x'
      - 'refs/pull/*'
      tags:
```

What I found (through a LOT of trial and error), was the following:

- I started with just pull_request. When I did that, additional pushes to the PR branch would not trigger checks.
- When I added "push", without qualifications, I would get TWO runs of checks on opening a PR, one for the PR, and one for the branch. This led to unnecessary builds.
- If I added branches to the mix (as I did) to qualify which branch triggered a build... tags would not build.

Basically, I found `on: [push, pull_request]` was building _too much_, which led me to qualify the values.

If you can point to repos where you have rolled out a workflow with the simplified entry, please link me to it so I can see the build history and make sure it's working the way I would want it to; until then, I'm a bit wary of this change.

Reply:
I've thought that `- '[0-9]+.[0-9]+.x'` might be too much for projects which do not follow the branch strategy required by the `laminas-automatic-releases` GHA.

Reply:
We could demonstrate two variants, then, maybe:

- The one we're using (and link to automatic-releases to detail the _why_ aspect)
- A more general one (and detail issues such as double-runs, if they're still valid)